src/Illuminate/Support/Facades/DB.php

```diff
@@ -60,10 +60,7 @@
  * @method static \Illuminate\Database\Connection setRecordModificationState(bool $value)
  * @method static void forgetRecordModificationState()
  * @method static \Illuminate\Database\Connection useWriteConnectionWhenReading(bool $value = true)
- * @method static bool isDoctrineAvailable()
  * @method static bool usingNativeSchemaOperations()
- * @method static \Doctrine\DBAL\Schema\Column getDoctrineColumn(string $table, string $column)
- * @method static \Doctrine\DBAL\Schema\AbstractSchemaManager getDoctrineSchemaManager()
  * @method static \Doctrine\DBAL\Connection getDoctrineConnection()
```

Comment:
@hafezdivandari you might still be working on this so sorry if I'm a bit too eager with mentioning this but it seems there's still some `\Doctribe\DBAL\` references in the source code remaining.

Reply:
@driesvints you want me to remove `getDoctrineConnection()`? then we can also remove the whole `/Illuminate/Database/PDO` directory, I kept this for the users/packages to be able to use doctrine connection, but we have no usage on the framework, maybe we can move these into another package and remove from core?

Reply:
I'll talk to the team about it.

Meanwhile I see `Illuminate\Database\Schema\Grammars\ChangeColumn` still using Doctrine. Could you maybe explain why you left this in? (not saying it should be removed)

Reply:
Thanks, please let me know.

I'm removing `Illuminate\Database\Schema\Grammars\ChangeColumn`, as we now have native support for SQLite too and there is no reason we keep using `doctrine/dbal` as an alternative option for modifying columns.

Reply:
@hafezdivandari let's attempt this in a different PR should this one get merged. Keeps it a bit more clean.